src/zino/api/server.py

```diff
@@ -0,0 +1,48 @@
+import logging
+from asyncio import AbstractEventLoop
+from typing import Optional
+
+from zino.api.legacy import Zino1ServerProtocol, ZinoTestProtocol
+from zino.api.notify import Zino1NotificationProtocol
+from zino.state import ZinoState
+from zino.statemodels import Event
+
+_logger = logging.getLogger(__name__)
+
+
+class ZinoServer:
+    """Represents the two asyncio servers that work in tandem to implement the Zino 1 legacy API:
+
+    Port 8001 is the text-based command interface.
+    Port 8002 is the text-based notification interface.
+    """
+
+    API_PORT = 8001
+    NOTIFY_PORT = 8002
```

Comment:
Are these also hardcoded in Zino 1? Curitz, and thus howitz, can be configured to work with other ports than this.

Reply:
Indeed, they are.